### PR TITLE
Add figure plotting real neurons

### PR DESCRIPTION
**Pull request by tathey1**
Added notebook that plots neuron traces

also replaced "axon" with "primary" https://github.com/neurodata/brainlit/issues/258

### CODE REVIEW

File: experiments/axon_geometry/notebooks/examine_neurons.ipynb [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+    "    sample_query = df.loc[(df['seg_id'] == i)]\n",
+    "    num_segments = len(sample_query.index)\n",
+    "    if num_segments == 0:\n",
+    "        continue\n",
+    "\n",
+    "    for row, measure in enumerate(measures):\n",
+    "        for col, clas in enumerate(classes.keys()):\n",
+    "            sample_query = df.loc[(df['seg_id'] == i) & (df['class'] == clas) & (df['measure'] == measure)]\n",
+    "            value = np.mean(sample_query[\"value\"].to_numpy())\n",
+    "            data[row, col] = value\n",
```
Comment (JacopoTeneggi) [inline experiments/axon_geometry/notebooks/examine_neurons.ipynb:173]:
```suggestion
    "        for col, class in enumerate(classes.keys()):\n",
    "            sample_query = df.loc[(df['seg_id'] == i) & (df['class'] == class) & (df['measure'] == measure)]\n",
    "            value = np.mean(sample_query[\"value\"].to_numpy())\n",
    "            data[row, col] = value\n",
```

File: experiments/axon_geometry/notebooks/examine_neurons.ipynb [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "plt.imshow(results)\n",
+    "plt.title('Counts of Neurons with Different Curvature/Torsion Orderings')\n",
+    "plt.xlabel('Torsion ordering')\n",
+    "plt.ylabel('Curvdature ordering')\n",
+    "plt.show()\n",
+    "print(results)"
```
Comment (JacopoTeneggi) [inline experiments/axon_geometry/notebooks/examine_neurons.ipynb:247]: Maybe remove print statement and add colorbar to heatmap?

**Review by JacopoTeneggi — CHANGES_REQUESTED** [review]
Looks good, minor changes

File: experiments/axon_geometry/notebooks/examine_neurons.ipynb (hunk 1 above)
Comment (tathey1) [reply experiments/axon_geometry/notebooks/examine_neurons.ipynb:173]: class is a keyword, so let's stick with clas?

File: experiments/axon_geometry/notebooks/examine_neurons.ipynb (hunk 2 above)
Comment (tathey1) [reply experiments/axon_geometry/notebooks/examine_neurons.ipynb:248]: added, although keeping the print statement so users can check it agrees with paper

**Review by JacopoTeneggi — APPROVED** [review]

**Pull request merged**